samples/nrf_desktop/src/services/hids.c

```diff
@@ -151,6 +156,26 @@ static void mplayer_notif_handler(enum hids_notif_evt evt)
 	notif_handler(evt, TARGET_REPORT_MPLAYER, REPORT_MODE_PROTOCOL);
 }
 
+static void feature_report_handler(struct hids_rep const *rep,
+				   struct bt_conn *conn)
+{
+	if (IS_ENABLED(CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE)) {
+
+		if (rep->size > 0) {
+			if (rep->size != REPORT_SIZE_USER_CONFIG) {
+				LOG_WRN("Unsupported report size");
+			}
```

Comment:
```
if (size != correct)
   warn();
else
   do();
```

Reply:
actually this can be the only check (i.e. you don't need to check for `size > 0`